source/serializer/index.rst

```diff
@@ -3,17 +3,9 @@ Text Serializers
 ================
 
 The lowest-level way to convert between Adventure's data and other formats 
-are serializers. Some serializers convert to standard formats, while others 
+are text serializers. Some serializers convert to standard formats, while others
```

Comment:
why? they're not called text serializers anywhere else

Reply:
The page heading is "Text Serializers" but I'm open to changing the heading.